src/cpp/plantgl/tool/util_mutex.h

```diff
@@ -0,0 +1,92 @@
+/* -*-c++-*-
+ *  ----------------------------------------------------------------------------
+ *
+ *       PlantGL: Plant Graphic Library
+ *
+ *       Copyright 1995-2003 UMR Cirad/Inria/Inra Dap - Virtual Plant Team
```

Comment:
Yiu may update Copyright year and affiliation...

Reply:
For all files I suppose. I will try to use an automatic tools for that.

Reply:
@pfernique has developed this type of tool (like headache)